bin/doppler-gestures.py

```diff
@@ -16,7 +16,7 @@
 from scipy import signal
 
 TONE = 20000
-WINDOW = 500
+WINDOW = 1000
```

Comment:
this particular change was not necessary, i just ended up using 1000 each time for the plotter view and 512 for ambiguity view, which works with chunk 256/512